src/mon/Monitor.cc

```diff
@@ -4400,8 +4404,13 @@ void Monitor::handle_ping(MonOpRequestRef op)
   boost::scoped_ptr<Formatter> f(new JSONFormatter(true));
   f->open_object_section("pong");
 
-  list<string> health_str;
-  get_health(health_str, NULL, f.get());
+  if (osdmon()->osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS) {
```

Comment:
i wonder if there is any daemon read the payload of `MPing`? can we just drop the code to stuff the reply?

Reply:
Yeah, no users.  There was clearly some intent but it isn't implemented.  @jecluis, any objection to just removing this?  @tchaikov he may still be on vacation, in which case let's leave this for now and follow up when he gets back?

Reply:
sure. then this change lgtm.